tex/latex/biblatex/lbx/latvian.lbx

```diff
@@ -384,9 +384,9 @@
   and              = {{un}{un}},
   andothers        = {{un citi}{u\adddotspace c\adddot}},
   andmore          = {{un t\={a} t\={a}l\={a}k}{utt\adddot}},
-  volume           = {{s\={e}jums}{s\={e}j\adddot}},
-  volumes          = {{s\={e}jumi}{s\={e}j\adddot}},
-  involumes        = {{s\={e}jumos}{s\={e}j\adddot}},
+  volume           = {{s\={e}jums}{s\={e}jums}},
+  volumes          = {{s\={e}jumi}{s\={e}jumi}},
+  involumes        = {{s\={e}jumos}{s\={e}jumos}},
```

Comment:
In that case "in 10 volumes" should get translated to "10 sējumos". But it looks like currently it would get wrongly translated to "10 sējumi" if `volumes` is used as the last word.
I guess it would indeed best to leave it untranslated.

Reply:
Unfortunately, the `.lbx` files often work under the assumption of English and German grammar constructions, so sometimes things just don't work. (See the date stuff.) Since this is a minor point and the string is never used in the standard styles, I think we can just leave it.

A long-term goal is to overhaul the entire `.lbx` system to allow for languages with grammar different from English and German, but I don't see this happening quite soon, and we would need to be careful approaching to not get stuck in the West-Germanic language mindset.